app/views/providers/search.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= content_for :page_title, "Organisation search" %>
+
+<h1 class="govuk-heading-xl">Pick a provider</h1>
+
+<ul class="govuk-list">
+  <%= render partial: "providers/provider_suggestion", collection: ProviderSuggestion.suggest(params[:query]) %>
```

Comment:
How many items are returned by the backend here? We should limit it to 5.

Reply:
Ah you're right I did forget to add a limit on the backend.